media-plugins/vdr-undelete/vdr-undelete-2.3.0_pre1.ebuild

```diff
@@ -1,18 +1,17 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
 
 inherit vdr-plugin-2
 
 DESCRIPTION="VDR Plugin: Recover deleted recordings of VDR"
 HOMEPAGE="http://phivdr.dyndns.org/vdr/"
 SRC_URI="http://phivdr.dyndns.org/vdr/vdr-undelete/vdr-undelete-2.3.1-pre1.tgz -> ${P}.tgz"
```

Comment:
Why is the version hardcoded here (and not corresponding to ${PV}, 2.3.1 vs 2.3.0)?

Reply:
Hi @thesamesam ,
I made a rebase with the current tree and found that you fixed this in October 2022. Your changes are back in here.